lib/plugins/aws/provider/awsProvider.js

```diff
@@ -321,6 +333,9 @@ class AwsProvider {
                 type: 'object',
               },
             },
+            // websockets properties
+            websocketsApiName: { type: 'string' },
+            websocketsApiRouteSelectionExpression: { type: 'string' },
```

Comment:
Let's also configure schema for `provider.apiGateway.websocketApiId` (Sorry, I know it was missing in issue, I've just added it)

Reply:
done!